packages/react-icons/package.json

```diff
@@ -44,10 +43,13 @@
     "react": "^17.0.1",
     "renamer": "^2.0.1",
     "svgo": "^1.3.2",
-    "tslib": "^2.1.0",
     "typescript": "~4.1.0",
     "yargs": "^14.0.0"
   },
+  "dependencies": {
+    "@griffel/react": "^1.0.0",
+    "tslib": "^2.1.0"
+  },
   "peerDependencies": {
     "@griffel/react": "^1.0.0",
```

Comment:
Can you please clarify why `@griffel/react` is still in `peerDependencies`? Is it intentional?